packages/components/src/components/hds/form/key-value-inputs/delete-row-button.ts

```diff
@@ -32,8 +32,13 @@ export default class HdsFormKeyValueInputsDeleteRowButton extends Component<HdsF
         this.args.onRemove();
       }
 
-      // move focus to the fieldset element so people using a keyboard are not forced back to the top of the page.
-      this.args.returnFocusTo?.focus();
+      const { returnFocusTo } = this.args;
+
+      if (returnFocusTo && document.body.contains(returnFocusTo)) {
```

Comment:
```suggestion
      if (returnFocusTo && (document.body.contains(returnFocusTo) || returnFocusTo.isConnected) {
```
[Question] Could the `else if` below this block be consolidate into one if statement?